docs/book/security.md

```diff
@@ -98,6 +98,12 @@ and provide to the authorization handler. When you use the `token`
 authentication, you must configure an authorization policy that checks the
 tokens. If the client does not supply a Bearer token, the `input.identity`
 value will be undefined when the authorization policy is evaluated.
+- Client TLS certificates: Client TLS authentication is enabled by starting
+OPA with ``-authentication=tls``. When this authentication mode is enabled,
+OPA will require all clients to provide a client certificate. It is verified
+against the CA certificated provided via `--tls-ca-cert-path`. Upon successful
+verification, the `input.identity` value is set to the TLS certificate's
+subject.
```

Comment:
I think an example to help people get started would be useful here.

Reply:
✔️ I've expanded the documentation. What do you think? 😃